admin/services/data-access/sql.service.js

```diff
@@ -394,4 +399,19 @@ sqlService.update = async function (tableName, data) {
   }
 }
 
+/**
+ * Helper function useful for constructing parameterised WHERE clauses
+ * @param {Array} ary
+ * @param {Tedious.TYPE} type
+ * @return {Promise<{params: Array, paramIdentifiers: Array}>}
+ */
+sqlService.whereClauseHelper = (ary, type) => {
+  const params = []
+  const paramIdentifiers = []
+  for (let i = 0; i < ary.length; i++) {
+    params.push({ name: `p${i}`, type, value: ary[i] })
+    paramIdentifiers.push(`@p${i}`)
+  }
+  return {params, paramIdentifiers}
+}
```

Comment:
nice 👌 
maybe consider naming it something more specific such as 'buildParameterList' or something similar?